src/main/java/org/ajoberstar/gradle/bintray/BintrayPlugin.java

```diff
@@ -1,13 +1,32 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.ajoberstar.gradle.bintray;
 
 import org.gradle.api.Project;
 import org.gradle.api.Plugin;
 
 public class BintrayPlugin implements Plugin<Project> {
-    @Override
-    public void apply(Project project) {
-        project.getPluginManager().withPlugin("maven-publish", plugin -> {
-            project.getPluginManager().apply(BintrayRules.class);
-        });
-    }
+  @Override
+  public void apply(Project project) {
+    project
+        .getPluginManager()
+        .withPlugin(
+            "maven-publish",
+            plugin -> {
+              project.getPluginManager().apply(BintrayRules.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 21. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AIndentationCheck)